common/app/model/meta.scala

```diff
@@ -414,12 +414,24 @@ case class TagCombiner(
   pagination: Option[Pagination] = None
 ) extends StandalonePage {
 
+  //We only use the left tag for CommercialProperties
+  val commercialPropertiesEditionAdTargeting: Seq[EditionAdTargeting] =
+    leftTag.properties.commercial.map(_.editionAdTargetings).getOrElse(Nil)
+  val commercialPropertiesEditionBrandings: Seq[EditionBranding] =
```

Comment:
Yeah, I moved them out because the inline code in `CommercialProperties` was even longer when using both the left and right tags. Can probably inline it again though without it looking too long.